### PR TITLE
bugfix/522

### LINKED ISSUE #522: restricted areas are not rendering

### PR DESCRIPTION
**Pull request by n8rzz**
resolves #522 

updates `AirportModel.buildRestrctedAreas` to build coordinate array with correct shape

### CODE REVIEW

File: src/assets/scripts/client/airport/AirportModel.js [hunk 1]
```diff
@@ -473,11 +473,8 @@ export default class AirportModel {
             }
 
             obj.height = parseElevation(area.height);
-            obj.coordinates = _map(area.coordinates, (v) => {
-                return [
-                    DynamicPositionModel.calculateRelativePosition(v, this._positionModel, this.magneticNorth)
-                ];
-            });
+            obj.coordinates = _map(area.coordinates,
+                (v) => DynamicPositionModel.calculateRelativePosition(v, this._positionModel, this.magneticNorth));
```
Comment (erikquinn) [inline src/assets/scripts/client/airport/AirportModel.js]: When doing this, don't you want the trailing `);` to wrap to the next line?

Comment (n8rzz) [inline src/assets/scripts/client/airport/AirportModel.js]: Mm, good catch. Yes. I'll fix that tonight after we get back from the farm.

Comment (n8rzz) [inline src/assets/scripts/client/airport/AirportModel.js]: done

File: CHANGELOG.md [hunk 2]
```diff
@@ -47,6 +47,8 @@
 - Runway, wind and spawnPattern changes to allow EGNM to operate [#492](https://github.com/openscope/openscope/issues/492)
 - Prevent attempts to access positions of vector waypoints [#467](https://github.com/openscope/openscope/issues/467)
 - Adjusts fix validation for hold/vector/flyover fix names [#451](https://github.com/openscope/openscope/issues/451)
+- Updates `AirportMode.buildRestrictedAreas()` to build the coordinate array with the correct shape [#522](https://github.com/openscope/openscope/issues/522)
```
Comment (erikquinn) [inline CHANGELOG.md]: `AirportModel`

**Pull request merged**